src/v1/sources/shopify/webpixelTransformations/pixelTransform.js

```diff
@@ -141,6 +141,11 @@ function processPixelEvent(inputEvent) {
   }
   message.anonymousId = clientId;
   message.setProperty(`integrations.${INTEGERATION}`, true);
+  message.setProperty('integrations.DATA_WAREHOUSE', {
+    options: {
+      jsonPaths: ['context.shopifyDetails'],
+    },
+  });
```

Comment:
Why are we setting these things in here? Are we expected to test something here? If yes then what?

Reply:
Is the same for other files as well?

Reply:
This is a legacy format where we don't specify event name in JSON paths.